plugin.audio.ilovemusic/changelog.txt

```diff
@@ -0,0 +1 @@
+v.1.0.0	- first Release
```

Comment:
Please also include this in a `<news>` tag in the addon.xml: https://kodi.wiki/view/Addon.xml